templates/layout/region--header-top.html.twig

```diff
@@ -13,8 +13,13 @@
  */
 #}
 {% if content %}
-  <div{{ attributes.addClass('header-top').setAttribute('id', 'hdbt-header-top').setAttribute('drupal-hdbt-selector', 'hdbt-header-top') }}>
-    <a href="https://hel.fi" class="link-to-site-frontpage">hel.fi</a>
-    {{ content }}
-  </div>
+	<div{{attributes.addClass('header-top').setAttribute('id','hdbt-header-top').setAttribute('drupal-hdbt-selector','hdbt-header-top')}}>
+		{% set link_attributes = {
+      'class': [
+        'link-to-site-frontpage',
+      ],
+    } %}
+		{{ link('hel.fi', 'https://hel.fi', link_attributes) }}
+		{{ content }}
```

Comment:
Indentation on this file seems to be all over the place 🙂

Reply:
Yes, there was tab indentation instead of spaces.